apps/web/src/routes/AllView/AllView.jsx

```diff
@@ -94,15 +88,23 @@ export default function AllViews () {
     }))
   }, [updateContextWidget])
 
-  const widgetsSorted = useMemo(() => {
-    return visibleWidgets.map(widget => {
-      return ContextWidgetPresenter(widget, { t })
-    }).sort((a, b) => a.title.localeCompare(b.title))
-  }, [visibleWidgets])
+  // Filter and sort widgets and get them ready for display
+  const visibleWidgets = useMemo(() => {
+    return contextWidgets.filter(widget => {
+      // When not editing only show widgets with a related view or chat room
+      if (!isEditing && !widget.view && widget.type !== 'chat') return false
```

Comment:
Just checking this is desired:
- Custom views that have been removed from the menu will be hidden from regular users in all way

Reply:
no we want those too. good catch